etc/docker/enterprise-gateway-demo/Dockerfile

```diff
@@ -4,6 +4,7 @@ ARG SPARK_VERSION
 FROM $HUB_ORG/demo-base:$SPARK_VERSION
 
 ENV NB_USER="jovyan"
+ENV SPARK_VER $SPARK_VERSION
```

Comment:
Same as previous.  Can we skip this?

I do think we will need to wrap $SPARK_VERSION as ${SPARK_VERSION} and I don't know if that only "available" on envs or can also be applied to ARGs.